app/controllers/api_entreprise/cas_usages_controller.rb

```diff
@@ -6,8 +6,9 @@ class APIEntreprise::CasUsagesController < APIEntrepriseController
   def index; end
 
   def show
-    @data = I18n.t("api_entreprise.cas_usages_entries.#{params[:uid]}", raise: true)
-  rescue I18n::MissingTranslationData
+    @cas_usage = CasUsage.find(params[:uid])
+    @other_cas_usages = CasUsage.all - [@cas_usage]
+  rescue I18n::MissingTranslationData, ActiveRecord::RecordNotFound
```

Comment:
```suggestion
  rescue ActiveRecord::RecordNotFound
```

A priori ici ça suffit